src/main/java/nextstep/subway/applicaion/JGraphPathFinder.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.subway.applicaion;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Path;
+import nextstep.subway.domain.Station;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.List;
+
+@Component
+public class JGraphPathFinder implements PathFinder {
+    @Override
+    public Path find(List<Line> lines, Station source, Station target) {
+        WeightedMultigraph<Long, DefaultWeightedEdge> subwayGraph = makeSubwayGraph(lines);
```

Comment:
```suggestion
        final var subwayGraph = makeSubwayGraph(lines);
        final var dijkstraShortestPath = new DijkstraShortestPath<>(subwayGraph);
```

자바 10부터 제공하는 var 키워드를 사용하면 코드를 간결하게 작성할 수 있겠네요.

Reply:
저는 개인적으로 어떤 타입인지 파악하기 어려운 느낌이라 잘안쓰긴 했는데ㅎㅎ 
현구님은 해당 키워드를 자주 사용하시는지 궁금합니다! 만약 사용하신다면 어떤 경우에 주로 사용하시나요??

Reply:
저는 최대한 활용하는 편이에요.
작성할 코드량이 줄어들고 다른 객체로 바꿀 때 변수의 타입은 바꾸지 않아도 되서 편리하더라구요.
그리고 객체를 생성할 때 타입이 써 있어서 크게 불편하진 않았던 것 같습니다.
위 코드처럼 메서드를 통해 변수를 설정할 때는 타입을 바로 파악하기 어려울 수 있겠네요.
이런 경우 메서드명에 타입이 드러나도록 하거나 ide의 도움을 받는 편입니다.